src/main/java/com/accantosystems/stratoss/vnfmdriver/model/alm/ExecutionAsyncResponse.java

```diff
@@ -20,17 +21,25 @@
     @ApiModelProperty(value = "Failure Details")
     private FailureDetails failureDetails;
     @ApiModelProperty(value = "Outputs")
-    private Map<String, String> outputs;
+    private Map<String, String> outputs = new HashMap<>();
```

Comment:
Should this be final now?
```suggestion
    private final Map<String, String> outputs = new HashMap<>();
```